frontend/src/services/mock.ts

```diff
@@ -24,4 +25,70 @@ export const mockUser: User = {
 export const mockWallet: IWallet = {
     id: 40,
     balance: 100,
-}
\ No newline at end of file
+}
+
+export const mockTransactions: ITransaction[] = [
```

Comment:
Data is incorrect. As I said, imagine if you have 10k transactions. It's insane to send it all from server to client. It's better to precompute results and send it to client.
This how it can look like, but feel free to change the schema for your needs:
```javascript
{
    "expenses": {
        "transactions": [
            {
                "destination": "Apple",
                "amount": 100,
                "share": 40
            },
            {
                "destination": "Amazon",
                "amount": 60,
                "share": 30
            },
            {
                "destination": "Slata",
                "amount": 30,
                "share": 20
            },
            {
                "destination": "Others",
                "amount": 20,
                "share": 10
            }
        ],
        "total": 210
    },
    "income": {
        "transactions": [
            {
                "source": "Top Up",
                "amount": 300,
                "share": 100
            }
        ],
        "total": 300
    }
}
```